src/controller/CHIPDeviceController.h

```diff
@@ -100,117 +95,28 @@ class DLL_EXPORT DevicePairingDelegate
     virtual void OnPairingDeleted(CHIP_ERROR error) {}
 };
 
-class DLL_EXPORT ChipDeviceController : public SecureSessionMgrDelegate,
-                                        public RendezvousSessionDelegate,
-                                        public PersistentStorageResultDelegate
+class DLL_EXPORT DeviceController : public SecureSessionMgrDelegate, public PersistentStorageResultDelegate
```

Comment:
So this thing is named `chip::Controller::DeviceController`?  Probably OK....

This class desperately needs documentation explaining what it does.